### ISSUE TITLE
UI overhaul

### ISSUE THREAD

**Issue by TheMonsterFromTheDeep** [TheMonsterFromTheDeep/vague#19]
...again....

I would like to start working on this project again - I've recently been inspired by some stuff I've been doing on Scratch (e.g. https://scratch.mit.edu/projects/142679949/) and I think it would be cool to work on this again.

However, the UI is not really all that great - it was fun to build, but it honestly isn't that useful. So I think I should use a more standard UI.

I would like a lot of it to be drag-n-drop.

Basically, my new idea is that I would eventually like to turn this into an animation software (maybe) and so basically how it will work is you can just drag-n-drop new objects / layers / whatever it happens to be into the world and begin working on them there.

I don't know how well that will work, but I think I should try to see.
